src/traversal/printRelayOSSQuery.js

```diff
@@ -64,12 +62,13 @@ function printRelayOSSQuery(node: RelayQuery.Node): PrintedQuery {
     queryText,
     'printRelayOSSQuery(): Unsupported node type.'
   );
+  const variables = {};
+  for (const [key, value] of variableMap) {
```

Comment:
The previously used `mapObject` function didn't work on a `Map`, so I'm just recreating the end result with this `for` logic. If you think this would be cleaner being in its own function, I can pull it out into its own function.